packages/components/Button/src/ButtonTokens.android.ts

```diff
@@ -9,68 +9,68 @@ export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = () =>
       width: '100%',
     },
     medium: {
-      padding: globalTokens.spacing.xs,
+      padding: globalTokens.size40,
       borderWidth: globalTokens.stroke.width10,
       iconSize: 16,
       focused: {
         borderWidth: 0,
-        padding: globalTokens.spacing.sNudge,
+        padding: globalTokens.size60,
       },
       hasContent: {
         minWidth: 96,
-        paddingHorizontal: globalTokens.spacing.m - globalTokens.stroke.width10,
+        paddingHorizontal: globalTokens.size100 - globalTokens.stroke.width10,
         hasIconAfter: {
-          spacingIconContentAfter: globalTokens.spacing.sNudge,
+          spacingIconContentAfter: globalTokens.size60,
         },
         hasIconBefore: {
-          spacingIconContentBefore: globalTokens.spacing.sNudge,
+          spacingIconContentBefore: globalTokens.size60,
         },
         focused: {
-          paddingHorizontal: globalTokens.spacing.m,
+          paddingHorizontal: globalTokens.size120,
         },
       },
     },
     small: {
-      padding: globalTokens.spacing.xs - globalTokens.stroke.width10,
+      padding: globalTokens.size40 - globalTokens.stroke.width10,
```

Comment:
This is specific style file for android, we should map XS to size80 in references here.

Reply:
This also applies to references for :
`globalTokens.spacing.s` (value 12), which should now be "`globalTokens.size120`"

Reply:
I have to update the platform specific mappings, currently everything is mapped to win32/mac. Will push changes.